src/engraving/dom/dynamic.h

```diff
@@ -129,6 +131,31 @@ class Dynamic final : public TextBase
 
     bool hasVoiceAssignmentProperties() const override { return true; }
 
+    enum class Grip {
```

Comment:
This enum seems unused

Reply:
Edit: now I see it _is_ used, but only outside this header file. That's perhaps not ideal. Would it be an idea to reuse the existing Grip enum? You could add `LEFT` and `RIGHT` as aliases for `START` and `END`.